src/components/editor/rubric-criteria-groups-editor.js

```diff
@@ -338,7 +330,7 @@ class RubricCriteriaGroupsEditor extends composeMixins(
 		}
 
 		let totalScoreText = null;
-		if (this._hasTotalScore) {
+		if (this.totalScore !== 'undefined') {
```

Comment:
Not sure if you need the quotes.

```suggestion
		if (this.totalScore !== undefined) {
```

Reply:
You do, since it gets converted to a string when getting passed down from the parent

Reply:
Or does it really need to look for the string of `undefined`?

Reply:
Could you not just change [this](https://github.com/Brightspace/d2l-rubric/blob/3a0bb9455b3990bcab49b89146e78457730cbf4c/src/components/editor/rubric-structure-editor.js#L392) to be `total-score="${ifDefined(this._totalScore)}"`. Would cause it to just not be sent if it's undefined.

Reply:
That seems to work, I can add it